application/assets/tilemaps/maps/desert/rules.tmx

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm sorry, can you explain what this file does? Does this let you automatically set those 'meta-tiles' over those pits?